openfisca_us/variables/usda/usda.py

```diff
@@ -0,0 +1,10 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class is_usda_elderly(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = "Is elderly per USDA guidelines"
```

Comment:
Add formula based on age and the parameter

Also add `label`

Reply:
added the formula and label flag, still confused as to what should be in label vs documentation

Reply:
label should be 3-4 words, documentation can be longer